src/pint_pal/__init__.py

```diff
@@ -0,0 +1 @@
+import pint_pal.checkin
```

Comment:
Using `__init__.py` allows me to just do `import pint_pal as pp` at the top of the notebook, and then access my function as `pp.checkin.shuffle_pars()`. I did that partly because it reduces the proliferation of lines like `import pint_pal.checkin as ci`, and partly for the "P-Pdot" pun.

Reply:
I'm envisioning `checkin.py` as a place to put anything that helps with the process of checking in a timing solution, so "auto-git utilities" would presumably be included, but other things that don't involve git directly could eventually go there too.